README.md

```diff
@@ -1,76 +1,76 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Space Travellers
 
-# Project Name
+> This is a React and Redux based single page application about EartQuake.  
 
```

Comment:
- [ ] Try to explain what the project is all about in the description and make sure you use the correct grammar like example its `EarthQuake` not `EartQuake`

Reply:
- [ ] I suggest that you add instructions in the README on how to use your application so that even anyone can understand how to use it

Reply:
Thank you for review